src/status_im/ui/screens/contacts/subs.cljs

```diff
@@ -26,9 +26,10 @@
   :all-added-contacts
   :<- [:get-contacts]
   (fn [contacts]
-    (->> (remove (fn [[_ {:keys [pending? whisper-identity]}]]
+    (->> contacts
+         (remove (fn [[_ {:keys [pending? mixable? whisper-identity]}]]
                    (or (true? pending?)
-                       (bots-constants/hidden-bots whisper-identity))) contacts)
+                       (true? mixable?))))
```

Comment:
Don't think you need `(true? ...)` here, as long as `mixable` is truthy or falsy, which it should be based on its name.

Reply:
You're right, it's just a leftover.